packages/account/src/Sections/Verification/ProofOfIncome/proof-of-income-form.tsx

```diff
@@ -0,0 +1,227 @@
+                                        </DesktopWrapper>
+                                        <MobileWrapper>
+                                            <SelectNative
+                                                name='document_type'
+                                                placeholder={localize('Select your document*')}
+                                                label={localize('Select your document*')}
+                                                value={values.document_type}
+                                                list_items={poinc_documents_list}
+                                                error={touched.document_type && errors.document_type}
+                                                use_text={true}
+                                                onChange={e => {
+                                                    setFieldValue('document_type', e.target.value, true);
+                                                    setStatus({ msg: '' });
+                                                }}
+                                                required
+                                                hide_top_placeholder
+                                            />
+                                        </MobileWrapper>
+                                    </React.Fragment>
+                                )}
+                            </Field>
+                        </fieldset>
+                        <div className='proof-of-income__form-field'>
+                            <FormSubHeader
+                                title={localize('Document submission')}
+                                title_text_size={is_mobile ? 'xs' : 's'}
+                            />
+                            <FileUploaderContainer
+                                onFileDrop={files => {
+                                    setDocumentFile(prevFiles => {
+                                        if (!isEqualArray(prevFiles, files)) setStatus({ msg: '' });
+                                        return files;
+                                    });
+                                }}
+                                onError={setFileSelectionError}
+                                files_description={
+                                    <FilesDescription
+                                        descriptions={files_descriptions}
+                                        title={files_descriptions_title}
+                                    />
+                                }
+                            />
+                        </div>
+                    </FormBody>
+                    <FormFooter>
+                        {status?.msg && <FormSubmitErrorMessage message={status.msg} />}
+                        <Button
+                            className='account-form__footer-btn'
+                            type='submit'
+                            is_disabled={
```

Comment:
Maybe it will be better to create a separate variable for this? And `document_file.length < 1` looks a bit weird:)
Like:
```
const is_disabled = isSubmitting || !isValid || !document_file.length  ||  !!file_selection_error || !!status.msg;
<Button ... is_disabled={is_disabled} ... />
```

Just asking, ignore if is not relevant.

Reply:
almost all variables are taken from Formik, and can’t be moved into outer scope.